amy/fiscal/tests/test_membership.py

```diff
@@ -172,8 +172,8 @@ def test_workshops_without_admin_fee(self):
         self.setUpTasks()
         self.assertEqual(self.current.workshops_without_admin_fee_per_agreement, 10)
         self.assertEqual(self.current.workshops_without_admin_fee_total_allowed, 10)
-        self.assertEqual(self.current.workshops_without_admin_fee_completed, 6)
-        self.assertEqual(self.current.workshops_without_admin_fee_planned, 4)
+        self.assertEqual(self.current.workshops_without_admin_fee_completed, 7)
+        self.assertEqual(self.current.workshops_without_admin_fee_planned, 3)
```

Comment:
Changes in the values are from the loosened constraints in calculation queryset.
6->7 was increased because event from outside membership date range is now counted in.
4->3 change was due to `completed` value increase (planned = available - completed).